Apps/Sandcastle/gallery/CZML Reference Properties.html

```diff
@@ -28,82 +28,80 @@
+        "label" : {
+            "fillColor" : {
+                "rgba" : [255, 255, 255, 255]
+            },
+            "font" : "13pt Lucida Console",
+            "horizontalOrigin" : "LEFT",
+            "outlineColor" : {
+                "rgba":[150, 0, 150, 255]
+            },
+            "outlineWidth" : 3,
+            "pixelOffset" : {
+                "cartesian2" : [20, 0]
+            },
+            "style" : "FILL_AND_OUTLINE",
+            "text" : "referencing position"
         },
-        "outlineWidth" : 3,
-        "pixelOffset" : {
-            "cartesian2" : [20, 0]
+        "position" : {
+            "reference" : "position-reference#position"
+        }
+    }, {
+        "id" : "polygon",
+        "name" : "Referencing Fill Color",
+        "description" : "<p>For more examples of reference properties, see CZML Polygon - Interpolating References.</p>",
+        "label" : {
+            "fillColor" : {
+                "rgba" : [255, 255, 255, 255]
+            },
+            "font" : "13pt Lucida Console",
+            "horizontalOrigin" : "LEFT",
+            "pixelOffset" : {
+                "cartesian2" : [20, 0]
+            },
+            "style" : "FILL_AND_OUTLINE",
+            "text" : "referencing fillColor"
         },
-        "style" : "FILL_AND_OUTLINE",
-        "text" : "referencing position"
-    },
-    "position" : {
-          "reference" : "position-reference#position"
-    }
-}, {
-    "id" : "polygon",
-    "name" : "Referencing Outline Color",
-    "description" : "<p>For more examples of reference properties, see CZML Polygon - Interpolating References.</p>",
-    "label" : {
-        "fillColor" : {
-            "rgba" : [255, 255, 255, 255]
+        "position" : {
+            "cartographicDegrees" : [-105, 35, 0]
         },
-        "font" : "13pt Lucida Console",
-        "horizontalOrigin" : "LEFT",
-        "pixelOffset" : {
-            "cartesian2" : [20, 0]
-        },
-        "style" : "FILL_AND_OUTLINE",
-        "text" : "referencing outlineColor"
-    },
-    "position" : {
-        "cartographicDegrees" : [-105, 35, 0]
-    },
-    "polygon" : {
-        "positions" : {
-            "cartographicDegrees" : [
-                -115.0, 37.0, 0,
-                -115.0, 32.0, 0,
-                -107.0, 33.0, 0,
-                -102.0, 31.0, 0,
-                -102.0, 35.0, 0
-            ]
-        },
-        "height" : 0,
-        "outline" : true,
-        "outlineColor" : {
-            "reference" : "outlineColor-reference#label.outlineColor"
-        },
-        "outlineWidth" : 5
-    }
-}];
+        "polygon" : {
+            "positions" : {
+                "cartographicDegrees" : [
+                    -115.0, 37.0, 0,
+                    -115.0, 32.0, 0,
+                    -107.0, 33.0, 0,
+                    -102.0, 31.0, 0,
+                    -102.0, 35.0, 0
+                ]
+            },
+            "height" : 0,
+            "fillColor" : {
```

Comment:
The fill color is not specified by `"fillColor"`, it'd be more like this:

```
"material" : {
            "solidColor" : {
                "color" : {
                    "reference" : "fillColor-reference#label.fillColor"
                }
            }
        }
```

See the CZML docs on polygons: https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Polygon